src/components/Typography/Typography.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, omitProps } from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-Typography');
+
+const { node, string, oneOf } = PropTypes;
+
+const Typography = createClass({
+	displayName: 'Typography',
+
+	statics: {
+		peek: {
+			description: `
+				A general component for several types of textual HTML elements.
+			`,
+			categories: ['text'],
+		},
+	},
+
+	propTypes: {
+		children: node`
+			Can contain elements or nested \`Typography\` components.
+		`,
+
+		className: string`
+			Appended to the component-specific class names set on the root element.
+		`,
+
+		variant: oneOf(['h1', 'h2', 'h3', 'p', 'a', 'pre', 'code'])`
+			This prop defines the type of text that will be displayed.
+			It may be an actual HTML element or something with extra semantic meaning.
+		`,
+	},
+
+	getDefaultProps() {
+		return {
+			variant: 'p',
+		};
+	},
+
+	elFinder: variant => {
+		switch (variant) {
+			case 'h1':
+				return 'h1';
+			case 'h2':
+				return 'h2';
+			case 'h3':
+				return 'h3';
+			case 'p':
+				return 'p';
+			case 'a':
+				return 'a';
+			case 'pre':
+				return 'pre';
+			case 'code':
+				return 'code';
+			case 'tabular':
```

Comment:
doesn't look like `tabular` is a variant prop.  If it's not a variant it doesn't seem like we need this fn?

Reply:
Also, I think [`<data>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/data) might be better for tabular data

Reply:
<data> could be the native element we use. I didn't want to use `data` as the key, just in case that we wanted to do something else with it.